groonga-apt-source/debian/changelog

```diff
@@ -1,3 +1,9 @@
+groonga-apt-source (2022.3.6-1) unstable; urgency=low
```

Comment:
This is invalid because this is a duplicated version.

I think that we can revert this change and reuse the current version because there is no `groonga-apt-source` for Ubuntu 22.04:

```bash
rake apt APT_TARGETS=ubuntu-jammy
```